src/components/input/range/index.d.ts

```diff
@@ -0,0 +1,19 @@
+import { ChangeEvent, FC, SyntheticEvent } from "react"
+
+export interface RangeInputProps {
+  "data-testid"?: string
+  max?: number
+  min?: number
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void
+  onClick?: (e: SyntheticEvent<HTMLButtonElement>) => void
+  onInput?: (e: SyntheticEvent<HTMLButtonElement>) => void
```

Comment:
Since we are implementing this component to replace the previous implementation I would have to say that it would be good to give the users the option to have the ones that we know at least that we are going to use! 😄